src/components/explore/ProportionExplorerView.svelte

```diff
@@ -95,12 +92,37 @@ h2 {
 
     <div class=body-control-set>
       <label class=body-control-set--label>Categories</label>
+      {#if bucketOptions.length > maxVisibleCategories}
+        <OptionMenu multi on:selection={(evt) => {
+          dispatch('selection', { selection: evt.detail.keys, type: 'activeBuckets' });
+        }}>
+          {#each bucketOptions as bucketOption, i (bucketOption)}
+            <Option
+              selected={activeBuckets.includes(bucketOption)}
+              key={bucketOption}
+              label={bucketOption}>
+              <!--
+                It's not currently possibly to specify a named slot for a
+                component, so we need to wrap this in a div.
+
+                https://github.com/sveltejs/svelte/issues/1037
+              -->
+              <div slot="right">
+                <ColorSwatch
+                  border="true"
```

Comment:
the black border makes this a bit hard on the eyes. Could we remove it?

Reply:
Keeping it in the component itself is a nice idea, but I would also _not_ ever use `black` for anything. `var(--cool-gray-900)` would be a better choice of a default.